test/Microsoft.AspNet.Mvc.Testing.Test/ControllerTestHelperTest.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
In this design PR, I didn't make these test more realistic. I'll do it in the actual PR.